codegen/Cargo.toml

```diff
@@ -9,11 +9,11 @@ version = "0.2.8"
 
 [build-dependencies.serde_codegen]
 optional = true
-version = "0.8.0"
+version = "0.8.9"
```

Comment:
I think this really ought to be `0.8.14`, as that's the version in the non-build dependencies for `serde_codegen` (and also the version that fixed #413).  I probably should have updated this as part of #414 but I missed it.